Two tests failing locally

Running `gulp test` locally leaves me with two failing tests:

```
  1) The emit processor Complains about bad time formatting in -points:
     AssertionError: expected undefined to include 'the time field must contain a number or a string representing time.'
      at test/runtime/specs/juttle-spec/procs/emit.spec.js:375:42
      at bound (domain.js:287:14)
      at runBound (domain.js:300:12)
      at tryCatcher (node_modules/bluebird/js/main/util.js:26:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/main/promise.js:507:31)
      at Promise._settlePromiseAt (node_modules/bluebird/js/main/promise.js:581:18)
      at Async._drainQueue (node_modules/bluebird/js/main/async.js:128:12)
      at Async._drainQueues (node_modules/bluebird/js/main/async.js:133:10)
      at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/main/async.js:15:14)

  2) The Date.parse function Parses a custom-formatted date:

      AssertionError: expected [] to deeply equal [ { winning: true } ]
      + expected - actual

      -[]
      +[
      +  {
      +    "winning": true
      +  }
      +]
      
      at Assertion.assertEqual (node_modules/chai/lib/chai/core/assertions.js:427:19)
      at Assertion.ctx.(anonymous function) [as equal] (node_modules/chai/lib/chai/utils/addMethod.js:40:25)
      at test/runtime/specs/juttle-spec/modules/date/parse.spec.js:47:50
      at bound (domain.js:287:14)
      at runBound (domain.js:300:12)
      at tryCatcher (node_modules/bluebird/js/main/util.js:26:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/main/promise.js:507:31)
      at Promise._settlePromiseAt (node_modules/bluebird/js/main/promise.js:581:18)
      at Async._drainQueue (node_modules/bluebird/js/main/async.js:128:12)
      at Async._drainQueues (node_modules/bluebird/js/main/async.js:133:10)
      at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/main/async.js:15:14)
```

This is with Node.js 4.2.4 installed via nvm. The failures don’t occur on Travis CI.

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "log4js": "^0.6.28",
     "marked": "0.3.3",
     "minimist": "1.1.0",
-    "moment": "^2.8.3",
+    "moment": "^2.11.1",
     "moment-duration-format": "^1.3.0",
     "moment-parser": "^0.3.0",
     "moment-timezone": "^0.4.0",

--- a/test/runtime/specs/juttle-spec/procs/emit.spec.md
+++ b/test/runtime/specs/juttle-spec/procs/emit.spec.md
@@ -384,7 +384,7 @@ Complains about bad time formatting in -points
 
 ### Juttle
 
-    emit -points [ {time:"1970-01-X01"} ]
+    emit -points [ {time:"invalid"} ]
     | view result
 
 ### Errors

